.devcontainer/Dockerfile

```diff
@@ -8,14 +8,18 @@ FROM mcr.microsoft.com/vscode/devcontainers/python:0-${VARIANT}
 ARG NODE_VERSION="none"
 RUN if [ "${NODE_VERSION}" != "none" ]; then su vscode -c "umask 0002 && . /usr/local/share/nvm/nvm.sh && nvm install ${NODE_VERSION} 2>&1"; fi
 
-RUN /usr/local/bin/python3 -m pip install --upgrade pip
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get -y install --no-install-recommends tig python2.7 python2.7-dev
 
+RUN /usr/local/bin/python3 -m pip install --upgrade pip
 COPY requirements.txt /tmp/pip-tmp/
 COPY dev-requirements*.txt /tmp/pip-tmp/
-RUN pip3 --disable-pip-version-check --no-cache-dir install -r /tmp/pip-tmp/dev-requirements-py3.txt && rm -rf /tmp/pip-tmp
-
-RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
-    && apt-get -y install --no-install-recommends tig python2.7 python2.7-dev
+RUN pip3 --disable-pip-version-check --no-cache-dir install -r /tmp/pip-tmp/dev-requirements-py3.txt \
```

Comment:
nit: can we break up the cmd for downloading dependencies for python3 (line 17) and python2.7 (line 18-22) and write a short comment to like `# Installing dependencies for python2.7` and `# Installing dependencies for python3.6`